RealmSwift/Sync.swift

```diff
@@ -314,12 +314,20 @@ public struct SyncConfiguration {
 public struct AppCredentials {
     /// An account token serialized as a string
     public typealias Token = String
+    /// An api key serialized as a string
+    public typealias APIKey = String
```

Comment:
This is fairly awkward, seems like having this as a `String` would be less confusing.